libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -1221,15 +1221,19 @@ void AP_TECS::update_pitch_throttle(int32_t hgt_dem_cm,
     // @Field: PErr: difference between estimated potential energy and desired potential energy
     // @Field: EDelta: current error in speed/balance weighting
     // @Field: LF: aerodynamic load factor
-    AP::logger().WriteStreaming("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF",
-                       "s------",
-                       "F------",
-                       "Qffffff",
+    // @Field: hdem1: demanded height input
+    // @Field: hdem2: rate-limited height demand
+    AP::logger().WriteStreaming("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,hdem1,hdem2",
+                       "s--------",
+                       "F--------",
+                       "Qffffffff",
```

Comment:
Shouldn't this be meters in the units format code?
You could also do the `* 0.01` using code `B` in the scaling bit but doesn't matter either way. 

```suggestion
    AP::logger().WriteStreaming("TEC2", "TimeUS,pmax,pmin,KErr,PErr,EDelta,LF,hdem1,hdem2",
                       "s------mm",
                       "F--------",
                       "Qffffffff",
```